### ISSUE TITLE
Saitek AV8R-01 - nothing gets through hid-remapper

### ISSUE THREAD

**Issue by knuf-kandler** [jfedor2/hid-remapper#99]
Sorry for the opening another problem, but maybe i am doing something fundamentally wrong and it will help the others.

I`ve found another joystick - Saitek AV8R-01 and i am trying to remap its axes and buttons to something recognizable by Android.

This time i can see all axes moving in web-configurator when joystick is connected to hid-mapper - X, Y, stick rotate, two throttles and all buttons are visible in monitor. 

But, when i try the most basic mapping (Cursor X - Right Stick X, Cursor Y - Right Stick Y), nothing gets through the remapper. 

Am I doing something wriong?

Attached - trace of clean Saitek, trace of Saitek through the mapper, config 

single-pico 

thx and rgds
[Saitek.zip](https://github.com/jfedor2/hid-remapper/files/13340300/Saitek.zip)

**Comment by jfedor2** [comment]
Yeah, it's not optimal, but currently if you want to map absolute axes you have to use expressions. So a mapping like:

Expression 1 -> Right Stick X

and Expression 1 looking something like this:
```
0x00010030 input_state
```
